dev/app/custom-widgets/jquery/currentRefinedValues.js

```diff
@@ -4,8 +4,8 @@ import instantsearch from '../../../../index.js';
 const renderFn = ({
   clearAllClick,
   clearAllURL,
-  clearRefinementURLs,
-  clearRefinementClicks,
+  createURL,
+  clearRefinement,
```

Comment:
Go for `refine()` see: https://github.com/algolia/instantsearch.js/pull/2107#discussion_r112231898